src/commands/list_folder.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+import { arweave } from 'ardrive-core-js';
+import { cliWalletDao, CLI_APP_NAME, CLI_APP_VERSION } from '..';
+import { ArDrive, ArDriveAnonymous } from '../ardrive';
+import { ArFSDAO, ArFSDAOAnonymous, ArFSPrivateFileOrFolderData, ArFSPublicFileOrFolderData } from '../arfsdao';
+import { CLICommand } from '../CLICommand';
+import { CommonContext } from '../CLICommand/common_context';
+import { ArDriveCommunityOracle } from '../community/ardrive_community_oracle';
+import {
+	DrivePasswordParameter,
+	ParentFolderIdParameter,
+	SeedPhraseParameter,
+	WalletFileParameter
+} from '../parameter_declarations';
+
+function alphabeticalOrder(a: string, b: string) {
+	return a.localeCompare(b);
+}
+
+new CLICommand({
+	name: 'list-folder',
+	parameters: [ParentFolderIdParameter, SeedPhraseParameter, WalletFileParameter, DrivePasswordParameter],
+	async action(options) {
+		const context = new CommonContext(options, cliWalletDao);
+		const wallet = await context.getWallet().catch(() => null);
+		const password = context.getParameterValue(DrivePasswordParameter);
+		const folderId = context.getParameterValue(ParentFolderIdParameter);
+		let children: (ArFSPrivateFileOrFolderData | ArFSPublicFileOrFolderData)[];
+
+		if (!folderId) {
```

Comment:
Is this already handled in the param declarations?

Reply:
Probably not worth a ticket, was just wondering if it was a redundant check.

Reply:
I think there could be a `getRequiredParameterValue()` that throws if the value isn't provided. 
That's not hard to implement, and it wouldn't be necessary to do that check every time we know it's present (because it's required)